### PR TITLE
Added Mission Control playbooks

### PR DESCRIPTION
**Pull request by kelby-shelton**
### Details

_What does this PR have in it? Screenshots are worth 1000 words 😄_

### Checklist

- [x] Validate name matches `<platform>_<mitre att&ck technique>_<short description>` nomenclature
- [x] [CI/CD](https://github.com/splunk/security_content/actions) jobs passed ✔️ 
- [x] Validated SPL logic.
- [x] Validated tags, description, and how to implement.
- [x] Verified references match analytic.

### CODE REVIEW

File: playbooks/Mission_Control_Attribute_Lookup.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: MC Attribute Lookup
+id: d76dd338-b2a0-4916-abdc-40dff5924c70
+version: 1
+date: '2023-03-14'
+author: Kelby Shelton, Splunk
+type: Investigation
+description: Launches "attribute lookup" input playbooks, creates events, and then concludes the Response Tasks where this playbook appears.
+playbook: MC_Attribute_Lookup
```
Comment (ljstella) [inline playbooks/Mission_Control_Attribute_Lookup.yml]: This needs to change to the following:
`Mission_Control_Attribute_Lookup`
Or else quite a few links will be broken.

File: playbooks/Mission_Control_Identifier_Reputation_Analysis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: MC Related Tickets Search
+id: 35c157a6-bfdc-44ef-8a67-8693a432caf5
+version: 1
+date: '2023-03-14'
+author: Kelby Shelton, Splunk
+type: Investigation
+description: Launches "related tickets" input playbooks, creates events, and then concludes the Response Tasks where this playbook appears.
+playbook: MC_Related_Tickets_Search
```
Comment (ljstella) [inline playbooks/Mission_Control_Identifier_Reputation_Analysis.yml]: This needs to change to:
`Mission_Control_Identifier_Reputation_Analysis`

File: playbooks/Mission_Control_Identifier_Reputation_Analysis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: MC Related Tickets Search
```
Comment (ljstella) [inline playbooks/Mission_Control_Identifier_Reputation_Analysis.yml]: This needs to be `Mission Control Identifier Reputation Analysis`

File: playbooks/Mission_Control_Related_Tickets_Search.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+name: MC Related Tickets Search
+id: 35c157a6-bfdc-44ef-8a67-8693a432caf5
+version: 1
+date: '2023-03-14'
+author: Kelby Shelton, Splunk
+type: Investigation
+description: Launches "related tickets" input playbooks, creates events, and then concludes the Response Tasks where this playbook appears.
+playbook: MC_Related_Tickets_Search
```
Comment (ljstella) [inline playbooks/Mission_Control_Related_Tickets_Search.yml]: Needs to match the filenames:
`Mission_Control_Related_Tickets_Search`

File: playbooks/Mission_Control_Related_Tickets_Search.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+name: MC Related Tickets Search
```
Comment (ljstella) [inline playbooks/Mission_Control_Related_Tickets_Search.yml]: Should be `Mission Control Related Tickets Search`

File: playbooks/Mission_Control_Attribute_Lookup.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+name: MC Attribute Lookup
```
Comment (ljstella) [inline playbooks/Mission_Control_Attribute_Lookup.yml]: Should be `Mission Control Attribute Lookup`

**Review by ljstella — COMMENTED** [review]
Couple of small issues:

The "Name" should really be the non-Camel/snake-cased version of the filename. Abbreviations like "MC" instead of "Mission Control" are inconsistent with other playbooks in which the asset name is used, and the asset name is tied to the app's formal name. 

The `playbook` field is to reference the other files within the directory and so needs to match them. 

The Identifier Reputation Analysis file includes a yaml that appears to just be the Related Tickets one over again, including the ID field.

**Comment by patel-bhavin** [comment]
moved to gitlab

**Pull request closed without merging**